server/queries/services/postServiceQuery.ts

```diff
@@ -0,0 +1,16 @@
+import connection from '../../database/connection';
+
+const postServiceQuery = (
+  name: String,
+  description: String,
+  price: number,
+  image: String,
+  categoryId: number,
+) => {
+  const sql = {
+    text: 'INSERT INTO services(name, description, price, image, category_id) VALUES ($1,$2,$3,$4,$5) RETURNING *',
```

Comment:
If you are not using the returned value then why would you return it in the first place?
```suggestion
    text: 'INSERT INTO services(name, description, price, image, category_id) VALUES ($1,$2,$3,$4,$5);',
```